### DISCUSSION TITLE
[Bug]: multi-select working with Controller can't submit data and do something onChange

### DISCUSSION THREAD

**Discussion by RenaudPavie** [react-hook-form/react-hook-form#6279] [Q&A]
### Version Number

6.15.5

### Codesandbox/Expo snack

https://codesandbox.io/s/rhf-react-select-multiple-07gpx

### Steps to reproduce

1. Select option in the first select and note that no options are displayed
2. Check the console to see the output onChange
3. Select option in the second select and note that options are displayed but nothing is output in the console
4(optional) . check some checkbox to fill the form
5. Submit and check the output object :
- No value for the first select
- Value for the second
- No problem for the rest of the form

### Expected behaviour

I'm using a multi select from React-Select in my form which use React Hook Form(RHF).
Basically i want to retrieve data when submit, but i also need to do some verification when something change in my form. The reason is that i have to limit the number of field the client can fill.
So i tried to attach an event on my form to handle every change in the form and it works for also all my fields but not for my multi-select which use Controller from RHF. 
Then i tried to attach a different handler on the select to do my verification and it works but now the data isn't sending anymore.
I searched and tried multiples things. I can submit the data, i can success to do the onChange but i cannot do both at the same time.

### What browsers are you seeing the problem on?

Chrome

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by bluebill1049** [comment]
```
<Controller
          render={({ value }) => (
            <Select
              options={[
                { value: "0", label: "test 1" },
                { value: "1", label: "test 2" },
                { value: "2", label: "test 3" },
                { value: "3", label: "test 4" }
              ]}
              value={value}
              isClearable
              isMulti
              closeMenuOnSelect={false}
              onChange={(e) => handleChange(e)}
            />
          )}
          name="select.thisIsMySelect"
          control={control}
        />
```
        
you didn't use the onChange from the Controller.

**Comment by RenaudPavie** [comment]
Thanks for your fast answer :)
Even when i use it, i can't have my onChange function, and the value when submit, i updated my sandbox.
Am i doing something wrong with the controller ?
